.github/workflows/pr-check-signed-commits.yml

```diff
@@ -1,9 +1,12 @@
 name: Check signed commits in PR 
-on: pull_request
+on: pull_request_target
 
 jobs:
   build:
     name: Check signed commits in PR 
+    permissions:
+      contents: read
+      pull-requests: write
```

Comment:
Are you sure we can grant this? [This table](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) seems to suggest it is capped at `read` for public forks.

Reply:
According to the `actions/labeler` docs, it should be supported when using `on: pull_request_target`:

> To resolve this issue, you can modify the on: section of your workflow to use [pull_request_target](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target) instead of pull_request (see example [above](https://github.com/actions/labeler#create-workflow)). This change allows the action to have write access, because pull_request_target alters the [context of the action](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target) and safely grants additional permissions

Reply:
Ah, I read over that part. Thanks for clarifying.